neuralop/data/transforms/data_processors.py

```diff
@@ -336,7 +338,7 @@ def __init__(
             self.in_normalizer = in_normalizer.to(self.device)
         if out_normalizer:
             self.out_normalizer = out_normalizer.to(self.device)
-        self.model = None
+        self.model = model
```

Comment:
Do we use the model in the processor?

Reply:
In distributed settings, the `MultigridPatching2D` takes the model and adds a hook to the parameters that averages gradients.